### ISSUE TITLE
Fix GitHub Contributors Action

### ISSUE THREAD

**Issue by rishabkumar7** [learntocloud/learn-to-cloud#156]
## Expected Behavior
Run Contributors action weekly to generate an SVG

**Comment by rishabkumar7** [comment]
I think this is the blocker
https://github.com/orgs/community/discussions/23064

**Comment by azgcloudev** [comment]
@rishabkumar7  Not sure, if this is the correct fix, but in my main branch I was able to get the action successfull by enabling the following in the settings of the repo:

<img width="877" alt="imagen" src="https://github.com/learntocloud/learn-to-cloud/assets/72358828/5d4ac1b7-5faf-43d6-95c1-ea4383a6aa11">

**Comment by madebygps** [comment]
@rishabkumar7 take a look when you can please

**Comment by azgcloudev** [comment]
@rishabkumar7 

So I was looking at the error, and it says that it needs a PR to make changes.

As per the action github, I was reading a issue, and the owner mentioned that it creates a PR using the default branch of the repo because of the github API.

When I repro it did worked for me, and my branch is not Protected, is staging protected?

**Comment by madebygps** [comment]
@rishabkumar7

**Comment by rishabkumar7** [comment]
@azgcloudev yeah, staging is protected hence it creates a PR and can't just merge it.

**Comment by madebygps** [comment]
@rishabkumar7  what are we doing about this?

**Comment by rishabkumar7** [comment]
@madebygps I am trying to see if there is a workaround. Will disable the action for now.
